scripts/compilation

```diff
@@ -12,6 +12,8 @@ from buildpack.stage import DOT_LOCAL_LOCATION
 
 BUILD_PATH = sys.argv[1]
 CACHE_PATH = sys.argv[2]
+DEPS_DIR = sys.argv[3]
```

Comment:
Do we need to add some documentation for these 2 parameters? Or ar they documented in the CF-buildpack?

Reply:
It's a directory called `deps` with extra CF components (https://docs.cloudfoundry.org/buildpacks/understand-buildpacks.html#supply-script) - Docker Buildpack don't use it, but the CF Buildpack needs this extra argument to point to a valid directory path: https://github.com/mendix/cf-mendix-buildpack/pull/516/files.

An empty path is enough for the CF Buildpack to continue working correctly, this is an internal workaround rather than a feature.

Reply:
Great, thanks.
Everything else looks good, in my opinion.